src/pages/components/data-table/usage.mdx

```diff
@@ -277,148 +280,312 @@ import {
+#### Condensed grid mode
 
-<div className="image--fixed">
+The data table can be used on a condensed grid, but care should be taken to avoid any unintentional relationships with other UI elements. Use a hybrid grid or a dissimilar background color to avoid the components blending in to each other.
 
-![Overflow menu example](images/data-table-usage-8.png)
+<Row>
+<Column colLg={12}>
 
-</div>
+![data table condensed ](images/datatable-alignment-condensed.png)
 
-<Caption>Overflow menu example</Caption>
+<Caption>Data table on a condensed grid</Caption>
 
-When the overflow menu contains less than three options, keep the actions inline as icon buttons instead. This reduces 1 click and makes available actions visible to users at a glance.
+</Column>
+</Row>
 
-<div className="image--fixed">
+## Behavior
 
-![Inline actions without overflow](images/data-table-usage-10.png)
+### Interactions
 
-</div>
+#### Mouse
 
-<Caption>Overflow menu example</Caption>
+The data table's toolbar and rows follow interaction conventions found in other Carbon components. Buttons, checkboxes, radio buttons, overflow menus, search, and expanding elements can be found in the data table and all behave the same way across Carbon components. See the style tab for detailed visual information about the various states.
 
-#### Persistent overflow menu
+<Row>
+<Column colLg={12}>
 
-By default, the overflow menu icon appears on hover or focus. This reduces visual clutter and drives user interactions with clear context and results.
+![data table target areas](images/datatable-target.png)
 
-Data table also supports persistent overflow menu icons. Overflow icons should only persist if the actions contained within them are core to the function of your table. Enabling persistent overflow icons may complicate data presentation and impact general usability.
+<Caption>Highlighted target areas for data table mouse interactions</Caption>
 
-<div className="image--fixed">
+</Column>
+</Row>
 
-![Persistent overflow icon example](images/data-table-usage-9.png)
+### Hover
 
-</div>
+A row's hover state can help your user visually scan the columns of data in a row. Consider leaving the hover state for your rows even if the row is not interactive.
 
-<Caption>Overflow menu example</Caption>
+<Row>
+<Column colLg={8}>
 
-### Expandable
+![data table row hover](images/datatable-hover.png)
+
+</Column>
+</Row>
+
+### Pagination
 
-The expandable data table is useful for presenting large amounts of data in a small space. Use the expanded section for supplementary information or data that needs additional query time.
+Pagination divides table data into separate pages. Pagination is accompanied by an option that enables the user to change the number of items per page. The pagination component is always placed at the bottom of the data table. See the [pagination](/components/pagination/code) component for further guidelines and configurations.
```

Comment:
"Advanced* pagination is accompanied by the option ... " You show two options but you don't talk about the two options.